src/core/qgsunittypes.cpp

```diff
@@ -1275,6 +1275,8 @@ QString QgsUnitTypes::encodeUnit( QgsUnitTypes::TemporalUnit unit )
       return QStringLiteral( "dec" );
     case TemporalCenturies:
       return QStringLiteral( "c" );
+    case TemporalIrregularStep:
+      return QStringLiteral( "xxx" );
```

Comment:
xxx, was that meant to be a temporary string prior to it being replaced by something better?

Reply:
No, it's deliberate. It's used when writing /reading the unit to projects and I wanted something which would never clash with any other units we may add in future :)